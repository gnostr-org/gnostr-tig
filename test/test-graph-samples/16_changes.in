commit 0000000000000000000000000000000000000000 0000000000000000000000000000000000000000
    Staged changes
commit 0000000000000000000000000000000000000000 007ae1f1c611254aaf67b71d42d2326be7ab4047
    Unstaged changes
commit 007ae1f1c611254aaf67b71d42d2326be7ab4047 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
    First commit
commit XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
    Last commit
