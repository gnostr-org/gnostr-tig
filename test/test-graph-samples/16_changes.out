● Staged changes
● Unstaged changes
● First commit
◎ Last commit
